nemo/collections/llm/README.md

```diff
@@ -0,0 +1,13 @@
+NeMo LLM Collection
```

Comment:
Please verify name. Which is correct: NeMo LLM Collection or NeMo Framework LLM Collection? Please make any changes as needed to the title.